src/providers/anthropic/chatComplete.ts

```diff
@@ -345,6 +345,22 @@ export const AnthropicErrorResponseTransform: (
   return undefined;
 };
 
+// this converts the anthropic stop_reason to an openai finish_reason
+const getFinishReason = (stopReason?: string): string | null => {
+  if (!stopReason) return null;
+
+  if (stopReason === 'max_tokens') {
+    return 'length';
+  }
+
+  if (stopReason === 'stop_sequence' || stopReason === 'end_turn') {
+    return 'stop';
+  }
+
+  // manages the case where the stop_reason is set but not recognized
+  return 'stop';
```

Comment:
This is okay, but not recommended, if we already know all the possible stop_reason supplied by anthropic, we should have a 1:1 mapping